config/config.go

```diff
@@ -703,3 +694,32 @@ func IsHostInetGateway() bool {
 	}
 	return false
 }
+
+// setFirewall - determine and record firewall in use
+func SetFirewall() {
+	if ncutils.IsLinux() {
+		if ncutils.IsIPTablesPresent() {
+			netclient.FirewallInUse = models.FIREWALL_IPTABLES
```

Comment:
What is the result of reporting FIREWALL_NONE if an ingress or egress is created? Will it be prevented, or just fail?

Reply:
Create Ingress will return error. Create Egress will not  (ticket NET-20)